fw/http_sched_ratio.c

```diff
@@ -24,6 +24,7 @@
 
 #include "sched.h"
 #include "tempesta_fw.h"
+#include "lib/alloc.h"
```

Comment:
Please rename `alloc.[ch]` to something not to confuse the simple error injection wrappers with rial allocation code, like `einj_alloc.[ch]`